src/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+	"GA_KEY": "Test-Google-Analytics-Key-Replace-Me"
```

Comment:
How do you replace the values doing the deployment. Can you provide more details?

Reply:
@longshuicy you're  trying to mount this config.json using helm's config map you said, right? But isn't this config.json needed for building process. If it is working with config map, then where should be the mounting point of the frontend container? I mean, where should I mount the config.json in the container that will be created after the building process. Or maybe I don't understand it correctly?